resources/helm/v2.1/istio-control/istio-discovery/files/gateway-injection-template.yaml

```diff
@@ -0,0 +1,234 @@
+{{- $containers := list }}
```

Comment:
We need to compare this with the sidecar injection template to pickup any changes we make there.